web/src/components/DeleteConfirm.js

```diff
@@ -57,7 +57,7 @@ class DeleteButton extends Component {
       <div>
         <Btn
           size="small"
-          extraCss="bg-black white h6"
+          extraCss="bg-black white h6 visibility--screen"
```

Comment:
Not really wild about this `extraCss` stuff here, but when we switch everything over to the CMS design system buttons, at least this can serve as documentation of which buttons should be hidden in print?

Reply:
👍